### DISCUSSION TITLE
How to issue a channel change voice command ?

### DISCUSSION THREAD

**Discussion by sharathbk1912** [andrei-tatar/node-red-contrib-smartnora#93] [General]
With new media player channel trait added , I have tried several commands but it doesnt work ?

Any suggestions  on a working voice command ?

Thanks and Regards
Sharath b K

**Comment by andrei-tatar** [comment]
I remember this one worked when testing this:
`hey google select channel 3 on TV`

It seems there's an issue in the schema when you select the channel by number. I will fix it shortly.

**Reply by andrei-tatar** [reply]
@sharathbk1912 Fixed the issue with the channel schema. Should work fine now... Just tested and `hey google select channel 3 on TV` works just fine for me.
